res/database/Default/database_scripts.xml

```diff
@@ -470,76 +470,582 @@
+						</character1>
+						<character2>
+							<de>ruhig und gelassen, immer einen kühlen Kopf bewahrend|bescheiden, trotz großen Talents bodenständig|analytisch, findet stets die besten Lösungen|charmant, mit einem Lächeln, das alle Türen öffnet|introvertiert und großartig im Zuhören|sanftmütig mit großer Geduld und Freundlichkeit|nachdenklich, betrachtet die Welt mit tiefem Verständnis|tolerant, mit großer Offenheit gegenüber anderen|empathisch, versteht intuitiv die Gefühle anderer|sehr sensibel, spürt die Emotionen anderer wie ein Seismograph</de>
+							<en>calm and composed, always keeps a cool head|modest, down-to-earth despite great talent|analytical, always finds the best solutions|charming, with a smile that opens all doors|introverted and great at listening|gentle with great patience and kindness|thoughtful, looks at the world with deep understanding|tolerant, with great openness to others|empathic, intuitively understands others' feelings|very sensitive, senses the emotions of others like a seismograph</en>
+						</character2>
+						<character3>
+							<de>ehrgeizig und entschlossen, setzt und erreicht hohe Ziele|sehr enthusiastisch, mit ansteckender Begeisterung|sehr organisiert mit beeindruckend präzisen Planungsfähigkeiten|begeisterungsfähig, findet Freude in den kleinen Dingen|intuitiv, liegt mit den Instinkten immer richtig|ambitioniert, strebt danach, die Welt zum Guten zu verändern|kreativ, die Ideen sprudeln nur so heraus, unerschöpflich|diplomatisch, schlichtet Konflikte und bringt Menschen zusammen|neugierig, saugt alles auf und entdeckt ständig neues|authentisch, bleibt immer sich selbst und verstellt sich nicht</de>
+							<en>ambitious and determined, sets high goals and achieves them|very enthusiastic, with contagious passion|very organized with impressively precise planning skills|enthusiastic, finds joy in the little things|intuitive, always has the right instincts|ambitious, strives to change the world for the better|creative, ideas bubble up inexhaustibly|diplomatic, mediates conflicts and brings people together|curious, absorbs everything and constantly discovers new things|authentic, always remains true and does not pretend</en>
+						</character3>
+					</variables>
 				</scripttemplate>
 				<scripttemplate index="1" product="2" licence_type="2" guid="scripttemplate-random-ron-averagedayseries01_02">
 					<title>
-						<de>%SUBJECT1A% im %SUBJECT1B%</de>
-						<en>%SUBJECT1A% in %SUBJECT1B%</en>
+						<de>%trouble% im ${location_${k}}</de>
+						<en>%trouble% in ${location_${k}}</en> <!-- TODO: apply capitalization function when available -->
```

Comment:
maybe you should already refactor the variables so that capitalization will not change the article but rather the location...

Reply:
definitely need both variations eventually